sample-compose/app/src/androidTest/java/co/nimblehq/sample/compose/ui/screens/home/HomeScreenTest.kt

```diff
@@ -0,0 +1,86 @@
+package co.nimblehq.sample.compose.ui.screens.home
+
+import androidx.activity.compose.setContent
+import androidx.compose.ui.test.*
+import androidx.compose.ui.test.junit4.ComposeContentTestRule
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.rule.GrantPermissionRule
+import co.nimblehq.sample.compose.domain.model.Model
+import co.nimblehq.sample.compose.domain.usecase.UseCase
+import co.nimblehq.sample.compose.test.CoroutineTestRule
+import co.nimblehq.sample.compose.ui.AppDestination
+import co.nimblehq.sample.compose.ui.screens.MainActivity
+import co.nimblehq.sample.compose.ui.theme.ComposeTheme
+import io.mockk.every
+import io.mockk.mockk
+import kotlinx.coroutines.flow.flowOf
+import org.junit.*
+import org.junit.Assert.assertEquals
+import org.junit.runner.RunWith
+
+@ExperimentalTestApi
+@RunWith(AndroidJUnit4::class)
```

Comment:
It seems we don't need to define this for `Compose` test. Please correct me If I missed something 🙏 
```suggestion
```

Reply:
@Wadeewee Good point, after checking, I found this `@RunWith is required only if you use a mix of JUnit3 and JUnit4.` https://developer.android.com/training/testing/instrumented-tests. So we don't need it in the template codebase.

Reply:
Fixed in https://github.com/nimblehq/android-templates/pull/384/commits/bda2a6a742ae459b455588e700ca4d387716ec29